master/security/icmp-ping.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Use ICMP/ping in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow and deny ICMP/ping messages. 
+
+### Value
+
+The **Internet Control Message Protocol (ICMP)** provides valuable network diagnostic functions, but it can also be used maliciously. Attackers can use it to learn about your network, or for DoS attacks. Using Calico network policy, you can control where ICMP is used. For example, you can:
+
+- Allow ICMP ping, but only within your cluster 
+- Allow ICMP for specifically labeled pods   
+  For example, allow ICMP for pods launched by operators for diagnostic purposes, but block other uses.
```

Comment:
I'm still a bit confused about this second list - what makes it need a separation from the one above?

I think maybe the two "For example" clauses without any other explanatory text between is throwing me off - are they both examples of the same thing? If so, shouldn't they be in the same list?

Reply:
The intention was a list of use case examples. I redid the list so maybe it reads more like it now. Better?